### ISSUE TITLE
Can not read CTA prod3b files

### ISSUE THREAD

**Issue by vuillaut** [cta-observatory/pyeventio#135]
Hi.
When trying to open files from CTA prod3b for La Palma gamma diffuse (e.g. `gamma_20deg_0deg_run100___cta-prod3-lapalma3-2147m-LaPalma_cone10.simtel.gz`) , I get an error (see below).

Is this a known error?

```
---------------------------------------------------------------------------
NotImplementedError                       Traceback (most recent call last)
<ipython-input-55-fcad501f1496> in <module>()
----> 1 source = SimTelFile(simtel_file)

~/anaconda/envs/cta-dev/lib/python3.5/site-packages/eventio/simtel/simtelfile.py in __init__(self, path, allowed_telescopes)
    134 
    135                 key = camel_to_snake(o.__class__.__name__)
--> 136                 self.telescope_descriptions[o.telescope_id][key] = o.parse()
    137 
    138         self._first_event_byte = self.tell()

~/anaconda/envs/cta-dev/lib/python3.5/site-packages/eventio/simtel/objects.py in parse(self)
    241 
    242     def parse(self):
--> 243         assert_exact_version(self, supported_version=1)
    244         self.seek(0)
    245         byte_stream = BytesIO(self.read())

~/anaconda/envs/cta-dev/lib/python3.5/site-packages/eventio/version_handling.py in assert_exact_version(self, supported_version)
      9                 name=self.__class__.__name__,
     10                 supported_version=supported_version,
---> 11                 given_version=self.header.version,
     12             )
     13         )

NotImplementedError: Unsupported version of CameraOrganization: only supports version 1got 2
```

**Comment by MaxNoe** [comment]
We never encountered a simtel file with camorgan v2 and we only implemented stuff we could test. 

Do you have a smallish file with camorgan v2? < 10mb?

You can check with the command line utility,  eventio_print_object_information

**Comment by vuillaut** [comment]
It seems that all files from prod3b are concerned (I just tested another one).
Here is what comes out of `eventio_print_object_information`

```
 Type | Version | #Objects | eventio-class
------------------------------------------------------------
   70 |       1 |        1 | eventio.simtel.objects.History
   71 |       1 |        1 | eventio.simtel.objects.HistoryCommandLine
   72 |       1 |    15470 | eventio.simtel.objects.HistoryConfig
  100 |       2 |        1 | eventio.histograms.Histograms
 1204 |       0 |     3005 | eventio.iact.objects.TelescopeData
 1208 |       2 |    43390 | eventio.iact.objects.PhotoElectrons
 1212 |       0 |        1 | eventio.iact.objects.InputCard
 2000 |       2 |        1 | eventio.simtel.objects.RunHeader
 2001 |       4 |        1 | eventio.simtel.objects.MCRunHeader
 2002 |       4 |      125 | eventio.simtel.objects.CameraSettings
 2003 |       1 |       86 | eventio.simtel.objects.CameraOrganization
 2003 |       2 |       39 | eventio.simtel.objects.CameraOrganization
 2004 |       2 |      125 | eventio.simtel.objects.PixelSettings
 2005 |       0 |      125 | eventio.simtel.objects.DisabledPixels
 2006 |       0 |      125 | eventio.simtel.objects.CameraSoftwareSettings
 2007 |       0 |      125 | eventio.simtel.objects.PointingCorrection
 2008 |       0 |      125 | eventio.simtel.objects.DriveSettings
 2009 |       2 |      459 | eventio.simtel.objects.TriggerInformation
 2010 |       0 |      459 | eventio.simtel.objects.ArrayEvent
 2011 |       2 |     5869 | eventio.simtel.objects.TelescopeEventHeader
 2013 |       3 |     5869 | eventio.simtel.objects.ADCSamples
 2014 |       5 |     5711 | eventio.simtel.objects.ImageParameters
 2015 |       1 |      431 | eventio.simtel.objects.StereoReconstruction
 2016 |       1 |     5869 | eventio.simtel.objects.PixelTiming
 2020 |       1 |    25000 | eventio.simtel.objects.MCShower
 2021 |       1 |   500000 | eventio.simtel.objects.MCEvent
 2022 |       0 |       86 | eventio.simtel.objects.CameraMonitoring
 2022 |       1 |       39 | eventio.simtel.objects.CameraMonitoring
 2023 |       2 |      125 | eventio.simtel.objects.LaserCalibration
 2026 |       2 |      459 | eventio.simtel.objects.MCPhotoelectronSum
 2027 |       0 |     7007 | eventio.simtel.objects.PixelList
 2100 |       0 |    57193 | eventio.simtel.objects.TrackingPosition
 2200 |       1 |     5869 | eventio.simtel.objects.TelescopeEvent
------------------------------------------------------------
```

I don't have a small file but I can send you one via secured channels.

**Comment by MaxNoe** [comment]
That really confuses me. It cannot be all prod3b files because I analyzed like 5 TB of prod3b files that @mackaiver downloaded to Dortmund.

Maybe it's just La Palma (why might that be?) or something else must be different. 

Anyways, @dneise or me will implement version 2 next week. Would be nice to have a small file for travis though.

**Comment by MaxNoe** [comment]
Can you check du ich telescopes have camorgan Version 2? 

Basically like this:

```
from eventio Import EventioFile
from eventio.simtel Import CameraOrganization

for o in EventioFile('path/to/file'):
    if isinstance(o, CameraOrganization):
        print(o.header.version, o.telescope_id)

```

**Comment by dneise** [comment]
> Would be nice to have a small file for travis though.

I would like to strengthen this statement. 
No feature without a test for it.
I cannot think of a test for this feature, without a file.

If it is difficult to generate a small file, one can generate a small file by just cutting into it. It should be enough if only one or two events survive. This copies the first 3000kB from `input_file_name` to `output_file_name`. 

    dd bs=1024 count=3000 if=input_file_name of=output_file_name

One will get some warnings when working with broken files like these but one will still be able to work...

**Comment by MaxNoe** [comment]
I could also do a clean cut. We also don't need any event as the camorgan is at the start of the file. I will check the file that Thomas send me via slack

**Comment by MaxNoe** [comment]
Ok, I implemented `CameraOrganisation` v2, no it seems we also need `CameraMonitoring` v1

NotImplementedError: Unsupported version of CameraMonitoring: only supports version 0got 1

**Comment by dneise** [comment]
Hello I implemented CameraMonitoring V1 in PR #138.

Tests are still missing, but maybe one would already like to play with that branch and see if it works in practice.

**Comment by dneise** [comment]
@vuillaut could you send me the file you send max here as well?

**Comment by vuillaut** [comment]
@dneise I share a link with you on the ctapipe Slack.

**Comment by MaxNoe** [comment]
Fixed in v0.16.1
